__tests__/plugin.test.ts

```diff
@@ -62,17 +62,32 @@ describe('babel plugin', () => {
     });
 
     it('injects its version', () => {
```

Comment:
Some of those tests could be moved to 'for debugging' section. Also we could add a test that checks for version not being present in release builds.